calico/scripts/install-calico-windows.ps1

```diff
@@ -166,11 +166,31 @@ function GetBackendType()
             throw "{{site.prodname}} on Linux has IPIP enabled. IPIP is not supported on Windows nodes."
         }
 
+        # Check FelixConfig first.
         $encap=c:\k\kubectl.exe --kubeconfig="$KubeConfigPath" get felixconfigurations.crd.projectcalico.org default -o jsonpath='{.spec.vxlanEnabled}' -n $CalicoNamespace
```

Comment:
Hm, the `-n $CalicoNamespace` seems odd to me since FelixConfigurations aren't a namespaced resource.

Reply:
I guess the `-n ...` is ignored. Fixed with [e10b910](https://github.com/projectcalico/calico/pull/6010/commits/e10b9109726e49da58c6e26ab36f8d401a2a8366)